tests/src/hipEnvVarDriver.cpp

```diff
@@ -81,12 +123,17 @@ void getDevicePCIBusNum(int deviceID, char* pciBusID) {
 }
 
 int main() {
-    unsetenv("HIP_VISIBLE_DEVICES");
-    unsetenv("CUDA_VISIBLE_DEVICES");
+    if (PLATFORM_NAME == "windows"){
+        _putenv("HIP_VISIBLE_DEVICES=");
+        _putenv("CUDA_VISIBLE_DEVICES=");
+    } else {
+        unsetenv("HIP_VISIBLE_DEVICES");
+        unsetenv("CUDA_VISIBLE_DEVICES");
+    }
```

Comment:
Add a new single define for _putenv/unsetenv in test_common.h. Here use it unconditionally.

Reply:
Hi, I would like clarification from something.
_putenv() and unsetenv() have slightly different arguments used here. Even if I define them, using them unconditionally won't work unless I also define the arguments too. What is preferred?

Reply:
Just use const declaration for `HIP_VISIBLE_DEVICES(=)` and `CUDA_VISIBLE_DEVICES(=)` under the same platform dependant conditional define.